packages/e2e/detox/hooks.js

```diff
@@ -0,0 +1,11 @@
+import { Before, After } from '@cucumber/cucumber';
+import detox from 'detox';
+
+Before({ timeout: 120 * 1000 }, async () => {
```

Comment:
This is auto-generated by detox and was planning to tweak it [and even see if it is really needed] when adding tests since not sure rn what the right numbers are.